src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1195,9 +1241,11 @@ public T onCompleted(Response response) throws Exception {
     }
 
     private static final String NEXT_PAGE = "next_page";
+    private static final String END_TIME = "end_time";
+    private static final long FIVE_MINUTES_IN_MS = TimeUnit.MINUTES.toMillis(5);
 
     private abstract class PagedAsyncCompletionHandler<T> extends AsyncCompletionHandler<T> {
-        private String nextPage;
+        String nextPage;
```

Comment:
does this really need escalation to `/*package*/` scope? if so, I'd prefer a getter/setter and keep it `private`